src/app/containers/Index/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { arrayOf, shape, node, bool } from 'prop-types';
+import frontPageGroup from '#models/propTypes/frontPageGroup';
+import { Grid, GridItemConstrainedLarge } from '#lib/styledGrid';
+import FrontPageSection from '../FrontPageSection';
```

Comment:
Should this be renamed to `IndexSection` if multiple indexes (indices?) are going to be using them in the future?

Reply:
Yeah I'd say so. I'm going to wait for more feedback around whether or not we should keep separate containers and then rename accordingly.